.github/workflows/e2e_test_ci.yml

```diff
@@ -180,1096 +180,1449 @@ jobs:
             spice
             spiced
 
-  test_quickstart_dremio:
-    name: 'Test Dremio quickstart (${{ matrix.target.target_os }}-${{ matrix.target.target_arch }})'
+  # test_quickstart_dremio:
+  #   name: 'Test Dremio quickstart (${{ matrix.target.target_os }}-${{ matrix.target.target_arch }})'
+  #   runs-on: ${{ matrix.target.runner }}
+  #   # run quickstart with external service dependency only on manual trigger
```

Comment:
I assume this is for testing? Can you mark this PR as draft and re-open when you are ready for a review

Reply:
@phillipleblanc - the change that uncomments temporary commented for testing tests, was not pushed, my bad, updated.